.github/scripts/validate_binaries.sh

```diff
@@ -15,8 +15,9 @@ else
         conda install -y conda=23.11.0
     fi
     # Please note ffmpeg is required for torchaudio, see https://github.com/pytorch/pytorch/issues/96159
-    conda create -y -n ${ENV_NAME} python=${MATRIX_PYTHON_VERSION} numpy ffmpeg
+    conda create -y -n ${ENV_NAME} python=${MATRIX_PYTHON_VERSION} ffmpeg
```

Comment:
Please restrict this change only for linux-aarch64 GPU builds. We want to continue testing the numpy from conda on all other builds

Reply:
Pushed a fix. Fingers crossed.